Makefile

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2017 SUSE LLC
```

Comment:
why did you updated the Makefile this way ? I wanted to start a PR to prepare the migration but this makefile can't build anything :(